rhel7/cpe/rhel7-cpe-dictionary.xml

```diff
@@ -62,6 +62,11 @@
             <!-- the check references an OVAL file that contains an inventory definition -->
             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_nss-pam-ldapd_package</check>
       </cpe-item>
+      <cpe-item name="cpe:/a:ntp">
+            <title xml:lang="en-us">Package ntp is installed</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_ntp_package</check>
+      </cpe-item>
```

Comment:
As the rule is applicable to Fedora, this same entry needs to be added to Fedora CPE dictionary too.